### PR TITLE
Add pull manager feature 

### PR DESCRIPTION
**Pull request by HansikaW**
This feature added to pull the images into the ukiyo agent, and the file bases approach has used to persist the credentials.

### CODE REVIEW

**Comment by HansikaW** [comment]
fixes #6

File: README.md [hunk 1]
```diff
@@ -28,3 +28,42 @@ Two modes of execution
...
+	 go mod init ukiyo
+	 go mod tidy
+
+### Create docker binary file
+	set GOARCH=amd64
+	set GOOS=linux
+	go build -ldflags="-s -w" -o ukiyo main.go
+
+### Docker build command
+	docker build -f Dockerfile -t agentukiyo/ukiyo .
```
Comment (rajikaimal) [inline README.md]: Add commands within markdown, example:
```suggestion
```
	docker build -f Dockerfile -t agentukiyo/ukiyo .
```
```

**Comment by rajikaimal** [comment]
@HansikaW can you update commands in README.md formatting

eg: 

```
docker build ...
```

**Comment by HansikaW** [comment]
Updated the README.md with formating.  @rajikaimal

**Pull request merged**